tests/sentry/plugins/sentry_webhooks/test_plugin.py

```diff
@@ -95,3 +95,17 @@ def test_webhook_validation(self):
 
         with pytest.raises(PluginError):
             validate_urls(form.cleaned_data.get("urls"))
+
+    @responses.activate
+    def test_moved_permanently(self):
+        """Test that we do not raise an error for 301s"""
+
+        responses.add(responses.POST, "http://example.com", body="<moved permanently", status=301)
```

Comment:
This response body is funny I like that it has to start with `<` so you tricked it like this 🔥

Reply:
Also 👏 we 👏 love👏 tests👏 , thanks for this!!